tests/test_core/test_alignment.py

```diff
@@ -1510,6 +1510,15 @@ def test_construction(self):
         seq2 = make_seq("AC", name="seq2")
         coll = SequenceCollection(data=[seq1, seq2])
 
+    def test_write_to_json(self):
+        # test writing to json file
+        aln = self.Class([("a", "AAAA"), ("b", "TTTT"), ("c", "CCCC")])
+        fn = mktemp(suffix=".json")
+        aln.write(fn)
+        result = load_unaligned_seqs(fn)
+        self.assertEqual(str(result), ">a\nAAAA\n>b\nTTTT\n>c\nCCCC\n")
```

Comment:
don't check str(result) since underlying structure based on dict and thus the record order is not guaranteed. Compare dict instead, i.e.

```python
self.assertEqual(result.to_dict(), dict(a="AAAA", ...)
```